server/api/src/rsDataObjUnlink.cpp

```diff
@@ -541,13 +541,17 @@ int dataObjUnlinkS(rsComm_t* rsComm,
             }
 
             if (!has_prefix(dataObjInfo->filePath, vault_path.data())) {
```

Comment:
We could just skip the whole block by checking here:
```suggestion
            if (!has_prefix(dataObjInfo->filePath, vault_path.data()) && UNREG_OPR != dataObjUnlinkInp->oprType) {
```
No need to set the oprType if it's already the right value. That said, what you have is fine.

Reply:
Updated.